src/components/Main/Burger/index.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import backGroundImg from "../../../assets/bg.jpg";
+import Loader from "../Loader";
+function Burger({ totalPrice, ingredientAddingOrder, loading }) {
+  const positionTopBun = ingredientAddingOrder.length
+    ? 115 + 25 * ingredientAddingOrder.length
+    : 260;
+  const zIndexTopBun = ingredientAddingOrder.length + 1;
+  return (
+    <div>
+      {loading ? (
+        <Loader />
+      ) : (
+        <Div>
+          <Tittle>Burger price: {totalPrice} ₴</Tittle>
+          <BurgerWrapper>
+            <TopBun
+              src={require("../../../assets/products/top_bun.png")}
+              alt="Top bun"
+              style={{
+                bottom: positionTopBun,
+                zIndex: zIndexTopBun,
+              }}
+            />
+            {!ingredientAddingOrder.length && (
+              <Paragraph>Start by adding ingredients to your burger</Paragraph>
+            )}
+
+            {ingredientAddingOrder.map((product, index) => {
+              return (
+                <ProductIMGStyled
+                  key={product + index}
```

Comment:
Better to avoid indexes in keys.
This can negatively impact performance and may cause issues with the component state.

https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/